shared/cmd/helpers.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrus.WithField("prefix", "node")
+
+// ConfirmAction uses the passed in actionText as the confirmation text displayed in the terminal.
+// The user must enter Y or N to indicate whether they confirm the action detailed in the warning text.
+// Returns a boolean representing the user's answer.
+func ConfirmAction(actionText string, deniedText string) (bool, error) {
+	var clearDB bool
+	reader := bufio.NewReader(os.Stdin)
+	log.Warn(actionText)
```

Comment:
Should this text be passed into the function and logged there or should we assume it to be logged from outside the function?

Reply:
Logging here is good